### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by Abhik1998**
Addressing to issue #12823

### CODE REVIEW

File: static/css/blog.css [hunk 1]
```diff
@@ -19,15 +19,15 @@ html body .region-inner {
 
 a:link {
   text-decoration:none;
```
Comment (daminisatya) [inline static/css/blog.css]: @Abhik1998 Thank you for the fix again!

This change will affect all the links which are supposed to be in blue. Refer the below screenshot/ verify on the Netlify deployment. 

screen shot 1940-12-07 at 1 47 19 pm

**Comment by tfogo** [comment]
Hi @Abhik1998, will you be able to make a fix for @daminisatya point above?

**Comment by kbhawkey** [comment]
Closing this PR as issue #12823 is closed. Feel free to reopen if needed.

**Pull request closed without merging**